src/pages/gaps/index.tsx

```diff
@@ -64,6 +39,32 @@ const GapsPage = () => {
   const [gapsIsLoading, setGapsIsLoading] = useState(false)
   const [onlyGapped, setOnlyGapped] = useSessionStorage('onlyGapped', false)
 
+  function formatTime(time: Moment) {
```

Comment:
Why did you move it inside the component?

Reply:
> Why did you move it inside the component?

I feel ashamed about why, and I am trying to think of another solution.  
You can call t / ``` const { t } = useTranslation() ``` outside of a react component because it is a hook. So I moved the functions that use t inside of the react component function. Very dirty, I need a shower now:)

Reply:
Oh, you're right. Its not dirty - I just didn't expect moving the time format to the translation file